models/models.js

```diff
@@ -31,3 +31,20 @@ exports.selectArticleById = (article_id) => {
       return result.rows[0];
     });
 };
+
+exports.selectCommentsById = (article_id) => {
+  return db
+    .query(
+      `SELECT comment_id, votes, created_at, author, body, article_id
+  FROM comments
+  WHERE article_id = $1
+  ORDER BY created_at DESC;`,
+      [article_id]
+    )
+    .then((result) => {
+      if (result.rows.length === 0) {
```

Comment:
Checking rows.length here is great but a valid article with no comments will _also_ reach this if statement, when actually it should be a 200, because nothing has gone wrong.

Reply:
You should use a checkExists before or after receiving empty rows